src/classes/UTIL_Describe_API.cls

```diff
@@ -292,5 +292,13 @@ global class UTIL_Describe_API {
     global static String getStudentCharacteristicAttributeRecType() {
         return UTIL_Describe.getStudentCharacteristicAttributeRecType();
     }
+
+    /*******************************************************************************************************
+    * @description Returns the list of all Child Relationship Name for a given Sobject Type
+    * @return List of all the child Relationship Names
+    */
+    global static List<String> getAllChildRelationshipNames(SobjectType sObjectTypeName) {
```

Comment:
@skosgi1 Please remove this global method. Though it does follow our former pattern, that is a pattern we are slowly moving away from. Let's handle wait to handle interoperability concerns on this method until we need to rather than in anticipation of needing to. Our pattern may be more mature at that time.